src/js/resources.js

```diff
@@ -1,9 +1,17 @@
 // Hard-coded URLs to resources of interest.
 // TODO: this could be driven by a separate domain.
+
 module.exports = {
-    etl: origins.endpoint + '/domains/pedsnet.dcc.loader/facts',
-    warehouse: origins.endpoint + '/domains/pedsnet.dcc.warehouse/facts',
-    pcornet: origins.endpoint + '/domains/pcornet.cdm/facts',
-    pedsnet: origins.endpoint + '/domains/pedsnet.cdm/facts',
-    i2b2: origins.endpoint + '/domains/i2b2.data/facts'
-};
+    models: ['pedsnet',
+             'pcornet',
+             'i2b2',
```

Comment:
For the data models service, the PEDSnet-specific i2b2 data model is named [`i2b2_pedsnet`](http://localhost:8123/models/i2b2_pedsnet/2.0.0). For now, adding a condition to use that name is sufficient, but we may need to think about a better way of defining the resources, such as a map from model to a set of endpoints across services.